lib/broccoli/ember-app.js

```diff
@@ -388,12 +385,8 @@ class EmberApp {
     @return
   */
   concatFiles(tree, options) {
-    this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.',
-      arguments[2] === SECRET_DEPRECATION_PREVENTION_SYMBOL);
-
-    options.sourceMapConfig = this.options.sourcemaps;
-
-    return concat(tree, options);
+    this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.');
```

Comment:
@rwjblue I guess once this lands we need to update the monkey patch until we ship the new fastboot build scheme right?